### ISSUE TITLE
can't work with Textarea?

### ISSUE THREAD

**Issue by xaero7** [inacho/bootstrap-markdown-editor#20]
like `<textarea id="editor">adsf</textarea>`

How to enable textarea?

**Comment by mageddo** [comment]
Hi, Today it really not works with `textarea`, I do not know why, you really need it?

**Comment by mageddo** [comment]
ping @xaero7

**Comment by inacho** [comment]
This plugin is built on top of [Ace editor](https://ace.c9.io) which uses a `<div>` element. That's the reason you need to put something like this in the submit event of your `form`

``` javascript
$('#myForm').submit(function(){
    var markdownContent = $('#myEditor').markdownEditor('content');
    $('#myTextarea').val(markdownContent);
});
```

**Comment by xaero7** [comment]
@mageddo Yes, I need it.
For my APP, it automaticly generates the form(including the textarea), and fill the form's value.
inacho gives a solution, but I think `bootstrap-markdown-editor` could work both for div and textarea.

**Comment by inacho** [comment]
I think I can improve the plugin to work with a `<textarea>` instead of a `<div>` and also sync the editor with the `<textarea>` automatically.
I'll keep you updated.

**Comment by inacho** [comment]
Released version 2.0.0
It works with a `<textarea>` instead of `<div>` and the text is synced automatically in the `<textarea>`.
